export const throwJsErrorInSyncProcessUseCase = () => {
  throw new Error('Error has occurred in synchronous process of EventHandler.');
};
